packages/graphql-server/src/functions/graphql.ts

```diff
@@ -342,30 +342,30 @@ const logResult =
 
     const options = {} as any
 
+    if (result.errors && result.errors.length > 0) {
+      envelopLogger.error(
+        {
+          errors: result.errors,
+        },
+        `GraphQL execution completed with errors:`
+      )
+    }
+
     if (result.data) {
       if (includeData) {
         options['data'] = result.data
       }
 
-      if (result.errors && result.errors.length > 0) {
```

Comment:
@callingmedic911  FYI the useRedwoodLogger refactoring in the next release also changes some the the error logging behavior, so may want to look at that change to see if it's approach is still ok:

https://github.com/redwoodjs/redwood/blob/247657b588d93ffa3490fd724c08e789a5eec54b/packages/graphql-server/src/plugins/useRedwoodLogger.ts#L121

```ts
  if (result?.errors && result?.errors.length > 0) {
      result.errors.forEach((error) => {
        if (
          error.originalError &&
          (error.originalError instanceof AuthenticationError ||
            error.originalError instanceof ForbiddenError)
        ) {
          envelopLogger.warn(
            {
              error,
            },
            `'${error?.extensions?.code || 'authentication'}' error '${
              error.message
            }' occurred in ${operationName}`
          )
        } else {
          envelopLogger.error(
            {
              error,
            },
            error.message || `Error in GraphQL execution: ${operationName}`
          )
        }
      })
    }
```

It warns on auth-related errors, and logs if other error type.

Reply:
Oh I didn't know, your changes seems to fix the issue already. Probably then we don't need this PR/changes unless we're doing another patch in between.